### DISCUSSION TITLE
Is there also Italian?

### DISCUSSION THREAD

**Discussion by MakerManu** [chromiecraft/chromiecraft#7355] [General]
Hello everyone and sorry to bother, I am simply here to ask if once I download the client and the game it results in Italian, both in dialog and voice acting.
The official game is, but I would like to know if it has been implemented here as well.
Thank you all!

PS. I am referring to a Wrath of the Lich King.

**Comment by heyitsbench** [comment]
It has not been implemented, no.

**Comment by Helias** [comment]
Ciao 👋 🇮🇹 

There is an unofficial patch for having Wrath of the Lich King in Italian, here you can find it https://shinworld.altervista.org/wordpress/world-of-warcraft-3-3-5a-in-italiano-ecco-la-patch/
If you need any help you can ping me on Discord ;-)

**Reply by MakerManu** [reply]
Thanks man!

**Reply by Helias** [reply]
prego ;-)
